### ISSUE TITLE
JS Errors When Attempting to Publish to Server from Actions menu in Content

### ISSUE THREAD

**Issue by Attackmonkey** [Jumoo/uSync.Complete.Issues#78]
**Describe the bug**
In the content section, you are unable to publish to other environments using the "Publish to server" button action.

When you click the "publish to server" action you get the popup to choose a server to publish to, and when you click the server from the list, the following JS error appears in the console:

```
angular.js?cdv=1:15635 TypeError: Cannot read property '0' of undefined
    at configController (configController.js?cdv=1:30)
    at Object.invoke (angular.js?cdv=1:5208)
    at $controllerInit (angular.js?cdv=1:11772)
    at nodeLinkFn (angular.js?cdv=1:10585)
    at compositeLinkFn (angular.js?cdv=1:9900)
    at publicLinkFn (angular.js?cdv=1:9765)
    at Object.link (angular.js?cdv=1:29889)
    at angular.js?cdv=1:1391
    at invokeLinkFn (angular.js?cdv=1:11334)
    at nodeLinkFn (angular.js?cdv=1:10653) "<div ng-include="vm.view.path" class="ng-scope">"
```

You ARE presented with the popup for send to server, and if you click the "send to server" button, you get the following JS error in the console and nothing happens:

```
angular.js?cdv=1:15635 TypeError: Cannot create property 'StepIndex' on string ''
    at setStep (overlayController.js?cdv=1:171)
    at setupAction (overlayController.js?cdv=1:149)
    at nextAction (overlayController.js?cdv=1:261)
    at Object.moveToNext (overlayController.js?cdv=1:296)
    at Object.submit (overlayManager.js?cdv=1:85)
    at ChildScope.scope.submitForm (umbraco.directives.js?cdv=1:8994)
    at fn (eval at compile (angular.js?cdv=1:16486), <anonymous>:4:220)
    at UmbButtonController.destination.<computed> [as action] (angular.js?cdv=1:11475)
    at UmbButtonController.clickButton (umbraco.directives.js?cdv=1:2191)
    at fn (eval at compile (angular.js?cdv=1:16486), <anonymous>:4:245)
```

Push/Pull content via the right click context menu in the tree works as expected, so it doesn't look like it's a config issue on the face of it.

**To Reproduce**
Steps to reproduce the behaviour:
1. Log into a site with uSync complete installed and one or more target servers set up
2. Click on a content node to edit it
3. Click on the arrow next to save and publish and selected "Publish to Server"
4. Select a server
5. First Error will appear
6. Click the "send to server" button
7. Second error will appear, nothing further happens

**Expected behavior**
When clicking the "publish to server" button, the content page should be published to the selected server.

**Desktop (please complete the following information):**
 - Windows & Mac OS
 - Browser Chrome, Safari, Firefox (Mac & PC)
 - Version: uSync Complete 8.8.0

**Additional context**
We have spotted this issue in uSync complete 8.8.0 on multiple sites. 

If you need any additional information, let me know and I'll see if I can dig it out for you!

**Comment by KevinJump** [comment]
Thanks, 

found that one now, we will wrap up a minor release with this (and any other current issues) in it. 

Kevin

**Comment by KevinJump** [comment]
Fixed in v8.8.1 (released now)
